app/views/roles/_form.html.erb

```diff
@@ -12,7 +12,26 @@
     <div class="tab-pane active" id="primary">
       <%= text_f f, :name, :class => @role.builtin? ? "disabled" : ""  %>
       <%= hidden_field_tag :original_role_id, @original_role_id if @cloned_role %>
+
+      <% if show_location_tab? || show_organization_tab? %>
+        <%= alert(:close => false, :class => 'alert-info', :header => '', :text => _("Changes to following associations will propagate to all inheriting filters ") +
+            popover("", _("Whenever %{orgs_or_locs} role association is changed, the change will propagate to all inheriting filters.
```

Comment:
Please use line breaks to format this, or even include it in the alert itself (it's ok to have long alerts)
![selection_014](https://cloud.githubusercontent.com/assets/598891/17997984/4695e8de-6b72-11e6-9ba1-1507e4c30b88.png)